resources/aro/working-in-multiple-clusters.md

```diff
@@ -0,0 +1,51 @@
+---
+tags:
+- context
+- oc config
+- multiple login
+- oc login
+- multi-cluster
+description: Learn how to more easily interact with two or more OpenShift Clusters using contexts
+---
+
+## Working in Multiple Clusters
+
+When performing a migration from one cluster to another it is important that you can rapidly change between clusters without having to 
+manually login. This can be done by leveraging `contexts`. Contexts allow you to change between clusters (when logged in) on the fly using the command `oc use-context <context name>`
```

Comment:
```suggestion
manually login. This can be done by leveraging `contexts`. Contexts allow you to change between clusters (when logged in) on the fly using the command `oc config use-context <context name>`
```